src/app_service/common/network_constants.nim

```diff
@@ -110,22 +110,24 @@ var NETWORKS* = %* [
     "isTest": false,
     "layer": 1,
     "enabled": true,
+    "relatedChainId": 5,
   },
   {
     "chainId": 5,
-    "chainName": "Goerli",
+    "chainName": "Mainnet",
```

Comment:
Does that impact the network selector?
Also what about edit network?

Reply:
Edit network will be done in a following task. not covered here

About the network filter, I found an issue which I am going to fix, but expected to work after that!

Reply:
Done @alaibe !